dump/dumpexfat.8

```diff
@@ -33,6 +33,11 @@ systems.
 Dump ranges of used sectors starting from 0 and separated with spaces. May be
 useful for backup tools.
 .TP
+.B \-f FILENAME
```

Comment:
Please also add this option to the SYNOPSIS section above. And I think it should be `file` instead of `FILENAME` for consistency with `dumpexfat` usage).